### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Eugene-Musika-cpamatica**
#### Features added:
* ```count()``` return quantity of elements in use
* ```elementComponentSize(<num>)``` except number of element and return size of his component
* ```disconnect(<num>[, <num>, ...])``` except list of elements to disconnect from components
* ```connect()``` and ```isConnected``` now can except more then 2 args

#### ⚠️  Attantion ⚠️
Test engine is changed from Should.js to Chai.js.
Execute ```npm install``` before starting tests.

### CODE REVIEW

**Comment by yuchiko** [comment]
Хорошо, что ты всё пытаешься сделать по своему, но так ты только усложняешь ревью

File: test/unionFind.spec.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const algolib = require('../index'),
+			expect = require('chai').expect;
+
+describe('#unionFind', function () {
+	it('should create propper data structure', function (done) {
```
Comment (pavlom0x) [inline test/unionFind.spec.js:7]: `done` help check asynchronous code, look like it's redundant here

File: test/unionFind.spec.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		expect(uf.isConnected(0, 3), '(0, 1)').to.be.true;
+
+		uf.connect(2, 8, 7);
+		expect(uf.isConnected(2, 7), '(2, 8, 7)').to.be.true;
+
+		done();
+	});
+
+
+	it('#count should return quantity of used elements', function (done) {
```
Comment (pavlom0x) [inline test/unionFind.spec.js:54]: actually `#count` should return the number of components

maybe it should have another name if it confused for example `componentCount`

**Review by pavlom0x — APPROVED** [review]
good job, I like your ES6

**Review by yuchiko — APPROVED** [review]

**Pull request merged**